### ISSUE TITLE
Cache orgs better

### ISSUE THREAD

**Issue by brandonhaslegs** [radicle-dev/radicle-upstream#2511]
Every time a user clicks on the orgs page, it loads a blank page, then orgs load a split second later. The layout should be cached and new orgs should be dynamically loaded onto the page when there are new ones available.

![orgs loading](https://user-images.githubusercontent.com/4406983/138439467-53e9a5a1-f71f-491c-9173-1649b438888e.gif)

Also I feel like orgs I've visited recently should also be cached. I've been visiting orgs repeatedly and every time I click on an org page (that I'd loaded already like 5 seconds ago) it has to load again. Since The Graph is unreliable and at best slow, let's just cache these so things load snappier!

![orgs loading low](https://user-images.githubusercontent.com/4406983/138457393-017c6a64-d03a-4b6d-b3b1-7ecad19f89cf.gif)

basically clicking back and forth to the orgs list and to an org and back to the orgs list and back to the same org should not result in any loading at all… it should all be cached. It's a waste of time to load it on every click.

**Comment by bordumb** [comment]
Came here to second this one.

I noticed this sort of behavior on the left-hand side of the main menu:
If I am added to a new Org, I have to entirely close/restart the Upstream app for it to load that Org membership.

Currently, it just has a gray square that says "loading Orgs..." but never loads, until I close/restart

This would be really nice to have!

**Comment by rudolfs** [comment]
> I noticed this sort of behavior on the left-hand side of the main menu: If I am added to a new Org, I have to entirely close/restart the Upstream app for it to load that Org membership.

As a workaround for now, you could press <kbd>⌘</kbd>+<kbd>r</kbd> to reload the app without quitting and reopening. This should clear the cache.

> Currently, it just has a gray square that says "loading Orgs..." but never loads, until I close/restart

Hm, that sounds like a bug. What steps get you into that state?

**Comment by bordumb** [comment]
FYI: saw this offline with @rudolfs on a screenshare

It seems random/intermittent and may be worth digging into the console logs (walletconnect/infura)

**Comment by Kaihuang724** [comment]
Closing this as orgs will not be implemented for MVP
